docs/features/wired-mesh.rst

```diff
@@ -71,7 +71,7 @@ When the client role is assigned to an interface in combination with other roles
 precedence, enabling mesh but not client in the previous example.
 
 The setup/config-mode interface is every interface with the role ``client`` which makes removing
-it from interfaces not only unnecessary, but generally unrecommended.
+it from interfaces not only unnecessary, but generally not recommended.
```

Comment:
Why would this be wrong?
https://en.wiktionary.org/wiki/unrecommended

```suggestion
it from interfaces not only unnecessary, but generally unrecommended.
```

Reply:
Try "unrecommended" with https://www.deepl.com/write. For me "not recommended" sounds better, but I am not a native English speaker.

Reply:
No, thanks.
https://en.wiktionary.org/wiki/unrecommended
https://www.dict.cc/englisch-deutsch/unrecommended.html
https://www.collinsdictionary.com/dictionary/english/unrecommended